### DISCUSSION TITLE
Github markdown to pdf conversion

### DISCUSSION THREAD

**Discussion by eoudejans** [ObjectVision/GeoDMS#66] [General]
Steps to export from github md to pdf on Fedora:
1) Install pandoc
2) Install texlive (full): dnf install texlive-scheme-full
3) Get git wiki project locally
4) Change image refs from github markdown: [[path/to/image]] to pandoc native: "![ ] ( path/to/image ) \
5) convert using pandoc command: pandoc -s -f markdown -t pdf -o test_geodms_wiki.pdf --pdf-engine=xelatex --trace --resource-path=./GeoDMS.wiki_edited/images ./GeoDMS.wiki_edited/Home.md

**Comment by mtbeek32** [comment]
After changing the syntax for the image refs, the images are not always loaded again, see attached screendump.

A refresh (F5) is needed to make the image visible.
![ImageNotLoaded](https://user-images.githubusercontent.com/24246911/211418539-42d09cf5-be65-4f3d-b636-dfc1e7aec583.png)

**Reply by eoudejans** [reply]
Issue noted, it seems Github can interpret the suggested image syntax, albeit in the second render pass. Which is why refreshing the page "solves" the image link visibility issue.

**Comment by eoudejans** [comment]
Internally pandoc converts to tex and then to pdf
